build.md

```diff
@@ -155,6 +155,35 @@ args:
   - GIT_COMMIT
 ```
 
+### ssh
+
+`ssh` defines an SSH authentication that the image builder SHOULD use during image build (e.g., cloning private repository)
+
+`ssh` property syntax can be either:
+* `default` - let the builder connect to the ssh-agent.
+* `ID=socket` - a key/value definition of an ID and the associated socket
+* `ssh` without any value which is a shortcut of the `default` configuration
```

Comment:
```suggestion
* `{}` (no value) which is a shortcut of the `default` configuration
```

Reply:
I suggest we use `{}` for empty as being used in a few other places

Reply:
How about using `true` instead? Seems clearer to me as a user but if `{}` is used in other places as well it might be more consistent.